pandas/_config/config.py

```diff
@@ -734,7 +734,7 @@ def config_prefix(prefix):
     global register_option, get_option, set_option, reset_option
 
     def wrap(func):
-        def inner(key: str, *args, **kwds) -> Callable:
+        def inner(key: str, *args, **kwds) -> Callable[[str, Any, Any], Any]:
```

Comment:
Really not sure if this is good.

@simonjayhawkins I would really appreciate if you take a look :)

Reply:
Are the `Any` supposed to represent args and kwargs? IIRC you don't need to annotate those

Reply:
> Are the `Any` supposed to represent args and kwargs? IIRC you don't need to annotate those

I was thinking the same, but because I annotated one argument (key) which is str, I suppose I must annotate all the other arguments.

otherwise I would annotate the function as:
 
```python
Callable[[str, ..., ...], ...]
```

Reply:
I don't think you need to annotate those. Probably better as just Callable[[str], Any] unless that doesn't work

Reply:
> I don't think you need to annotate those. Probably better as just Callable[[str], Any] unless that doesn't work

@WillAyd It worked!